docs/getting-started/go.mdx

```diff
@@ -160,7 +158,7 @@ func main() {
 	}
 }
 
-func run(ctx context.Context) error {
+func example(ctx context.Context) error {
 	listener, err := ngrok.Listen(ctx,
 		config.HTTPEndpoint(
```

Comment:
This is now out of date - the [config package was completely removed](https://github.com/ngrok/ngrok-go/pull/203/files#diff-f6d92a8f39b323b40cde569ed5f78ba3e1704e2c5976fa7b9046f8b093fef485).

Reply:
This will now look something like [this](https://github.com/ngrok/ngrok-go/blob/0aa328ac73f7169e57739e7959fb10e88dbd28ec/examples/traffic-policy/main.go)